os/logm/logm_process.c

```diff
@@ -42,10 +42,12 @@ static int logm_change_bufsize(int buflen)
 	}
 
 	/* Realloc new buffer with new length */
-	g_logm_rsvbuf = (char *)realloc(g_logm_rsvbuf, buflen);
-	if (!g_logm_rsvbuf) {
+	char *new_g_logm_rsvbuf = (char *)realloc(g_logm_rsvbuf, buflen);
+	if (new_g_logm_rsvbuf == NULL) {
+		dbg("Realloc Fail\n");
```

Comment:
How about change dbg to wdbg? I don't think this is enough important to print twice about failure. CONFIG_DEBUG can be used easily.